enrichments/trace/internal/elastic/span.go

```diff
@@ -237,6 +238,39 @@ func (s *spanEnrichmentContext) enrichTransaction(
 	}
 }
 
+// In OTel a root span can represent an outgoing call or a producer span
```

Comment:
```suggestion
// In OTel, a local root span can represent an outgoing call or a producer span
```

Reply:
Took the change, but moved this comment.